content/manual-references.json

```diff
@@ -941,5 +941,38 @@
     ],
     "type": "webpage",
     "standard_citation": "url:https://nihcc.app.box.com/v/ChestXray-NIHCC"
-  }
+  },
+{
```

Comment:
Missing indent. Not sure how you're editing this file, but a linter (built into many text editors) will really help you in the future (;